testsuite/features/secondary/proxy_as_pod_basic_tests.feature

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped:
+# * if there is no proxy ($proxy is nil)
+# * if there is no salt minion ($sle_minion is nil)
+# * if there is no scope @scope_containerized_proxy
```

Comment:
```suggestion
# The scenarios in this feature are skipped if:
# * there is no proxy ($proxy is nil)
# * there is no salt minion ($sle_minion is nil)
# * there is no scope @scope_containerized_proxy
```
I would allign this with the other .feature file and move the if to the first line.